### ISSUE TITLE
Open vs Closed issues per user

### ISSUE THREAD

**Issue by Schniz** [Schniz/colonel#2]
Can be a small number at the bottom of the profile picture.
Green circle on the right bottom for the closed number
Red circle at the left bottom for open tasks.

A profile picture with the numbers hidden, will have a line at the top/bottom that symbols the percentage of the open tasks.

That's for making the open tasks the main issue for the guy.
Maybe the percentage line could be at the top of the page. Very subtle and thin but effective.

**Comment by deanshub** [comment]
I work with trello, have a look at it's style
![](http://i.imgur.com/KYI1L3H.png)

**Comment by Schniz** [comment]
![image](https://cloud.githubusercontent.com/assets/2054772/5906397/f0ad31a0-a59f-11e4-9808-2ea5fd2b70d5.png)
Seems alike, but somehow different. @udidoron and I thought about the way to know how to categorize the issues

**Comment by Schniz** [comment]
the users on the right side are the team members, and they can be dragged on each issue(/task) and get assigned to the task by doing so.

**Comment by deanshub** [comment]
After everything is done, I suggest integrating a chat (+hubot) it's the ultimate place

**Comment by Schniz** [comment]
A chat for the project?

**Comment by deanshub** [comment]
yes, chat per project like gitter, that's a good way to hello people collaborate

**Comment by Schniz** [comment]
could be very nice! open an issue

**Comment by deanshub** [comment]
https://userstyles.org/styles/101224/trello-clear

**Comment by Schniz** [comment]
looks like OSX :smirk:
